openpype/hosts/tvpaint/plugins/load/load_workfile.py

```diff
@@ -0,0 +1,77 @@
+import getpass
+import os
+
+from avalon.tvpaint import lib, pipeline
+from avalon import api, io
+
+from openpype import Anatomy
+
+
+class LoadWorkfile(pipeline.Loader):
+    """Load workfile."""
+
+    families = ["workfile"]
+    representations = ["tvpp"]
+
+    label = "Load Workfile"
+
+    def load(self, context, name, namespace, options):
+        filepath = self.fname.replace("\\", "/")
+
+        if not os.path.exists(filepath):
+            raise FileExistsError(
+                "The loaded file does not exist. Try downloading it first."
+            )
+
+        george_script = "tv_LoadProject '\"'\"{}\"'\"'".format(
+            filepath
+        )
+        lib.execute_george_through_file(george_script)
+
+        # Save workfile.
+        host = api.registered_host()
+
+        project = io.find_one({
+            "type": "project"
+        })
+        session = api.Session
```

Comment:
You should use context of current workfile using `avalon.tvpaint.get_current_workfile_context` as `api.Session` may be set for different workfile.

Reply:
Done